### PR TITLE
Update lang id-ID

### PR DESCRIPTION
**Pull request by shiazza**
- Added some words 
- And changed a few typo

### CODE REVIEW

**Comment by shiazza** [comment]
Changed a few mistakes and added another language to make it easier to understand

**Comment by d2dyno1** [comment]
Can you add your agreement to #7625?

**Comment by shiazza** [comment]
> Bisakah Anda menambahkan perjanjian Anda ke #7625?



> Can you add your agreement to #7625?

I can

File: src/Files/MultilingualResources/Files.id-ID.xlf [hunk 1]
```diff
@@ -3369,155 +3369,155 @@ Kami menggunakan App Center untuk melacak penggunaan aplikasi, menemukan bug, da
         </trans-unit>
         <trans-unit id="ThirdPartyLicenses" translate="yes" xml:space="preserve">
           <source>Third party licenses</source>
-          <target state="new">Third party licenses</target>
+          <target state="new">Lisensi Pihak ke tiga</target>
```
Comment (d2dyno1) [inline src/Files/MultilingualResources/Files.id-ID.xlf:3372]: Please change the `state="new"` to `state="translated"` and in other places too!

File: src/Files/MultilingualResources/Files.id-ID.xlf [hunk 2]
```diff
@@ -208,7 +208,7 @@
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutNewFile.Text" translate="yes" xml:space="preserve">
           <source>File</source>
-          <target state="translated" state-qualifier="tm-suggestion">File</target>
+          <target state="translated" state-qualifier="tm-suggestion">Berkas</target>
```
Comment (d2dyno1) [inline src/Files/MultilingualResources/Files.id-ID.xlf:211]: Could you remove `state-qualifier="tm-suggestion"` and in other places too

**Review by d2dyno1 — CHANGES_REQUESTED** [review]

**Comment by shiazza** [comment]
>[d2dyno1] Reply

Done All completed

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**